libfwupd/fwupd-security-attr.c

```diff
@@ -865,6 +865,7 @@ fwupd_security_attr_to_json(FwupdSecurityAttr *self, JsonBuilder *builder)
 	}
 }
 
+
```

Comment:
remove this pls

Reply:
I had cleaned up the unnecessary changes.